application/parser/remote/reddit_loader.py

```diff
@@ -4,24 +4,22 @@
 
 class RedditPostsLoaderRemote(BaseRemote):
     def load_data(self, inputs):
-        client_id = inputs.get("client_id")
-        client_secret = inputs.get("client_secret")
-        user_agent = inputs.get("user_agent")
-        categories = inputs.get("categories", ["new", "hot"])
-        mode = inputs.get("mode", "subreddit")
-        search_queries = inputs.get("search_queries")
+        data = eval(inputs)
+        client_id = data.get("client_id")
+        client_secret = data.get("client_secret")
+        user_agent = data.get("user_agent")
+        categories = data.get("categories", ["new", "hot"])
+        mode = data.get("mode", "subreddit")
+        search_queries = data.get("search_queries")
         self.loader = RedditPostsLoader(
             client_id=client_id,
             client_secret=client_secret,
             user_agent=user_agent,
             categories=categories,
             mode=mode,
             search_queries=search_queries,
+            number_posts=10,
```

Comment:
Lets make sure users have an option to pass this as a parameter

Reply:
sure, i've made the change for that now